### ISSUE TITLE
Create plugin for emacs

### ISSUE THREAD

**Issue by suchow** [amperser/proselint#37]

**Comment by unhammer** [comment]
http://unconj.ca/blog/linting-prose-in-emacs.html defines one; I modified the regex a bit since errors can have newlines in them:
```
  (flycheck-define-checker proselint
    "A linter for prose."
    :command ("proselint" source-inplace)
    :error-patterns
    ((warning line-start (file-name) ":" line ":" column ": "
              (id (one-or-more (not (any " "))))
              (message (one-or-more not-newline)
                       (zero-or-more "\n" (any " ") (one-or-more not-newline)))
              line-end))
    :modes (text-mode markdown-mode gfm-mode))
  (add-to-list 'flycheck-checkers 'proselint)
```

**Comment by suchow** [comment]
Closed by #287.
